### PR TITLE
fix: fatal error when api-key not setup

### PR DESCRIPTION
**Pull request by alaminfirdows**
Initially when the plugin installed but not setup the API key yet, it throws an error. This PR fixes that fatal error.

image

### CODE REVIEW

File: src/Assistants/Assistant.php [hunk 1]
```diff
@@ -55,7 +55,7 @@ public static function instance(): Assistant
 
     public function load_assistant_script()
     {
-        $assistant_id = get_td_helpdesk_options()['td_helpdesk_assistant_id'];
+        $assistant_id = get_td_helpdesk_options()['td_helpdesk_assistant_id'] ?? '';
         if (empty($assistant_id)) {
             return;
         }
```
Comment (Rijoanul-Shanto) [inline src/Assistants/Assistant.php:61]: can you add an extra line between each statement?

Also, in lines 73 and 74, `wp_get_current_user()` was called twice, please normalize that.

**Review by Rijoanul-Shanto — CHANGES_REQUESTED** [review]
Minor changes

File: src/Assistants/Assistant.php (hunk 1 above)
Comment (alaminfirdows) [reply src/Assistants/Assistant.php:61]: it was just a quick fix for the error 😃

Comment (alaminfirdows) [reply src/Assistants/Assistant.php:62]: I've updated.

Comment (Rijoanul-Shanto) [reply src/Assistants/Assistant.php:62]: 💯

**Pull request merged**